troubleshooting.md

```diff
@@ -726,3 +726,23 @@ And then re-add the connection (removing the old one if necessary):
 And now this should work:
 
 `podman-remote info`
+
+---
+### 28) CNI networking fails in RHEL with Podman v2.2.1 to v3.0.1.
+
+A failure is encountered when trying to use networking on a rootless
+container in Podman v2.2.1 through v3.0.1 on RHEL.  This error does not
+occur on other Linux Distributions.
+
+#### Symptom
+
+Two rootless containers are created on the host but are not able to communicate
+with each other via their virtual network devices.
```

Comment:
I would suggest "A rootless container is created using a CNI network, but the `podman run` command returns an error that an image must be built"

Reply:
I've made your line the headliner in the symptom section, but said "Additionally..." and kept the old line as it was part of the original problem reports.

Reply:
This won't cause a failure when two containers are already created - this completely blocks the creation of containers in a CNI network until the image is built.